solutions/devsprint-riccardo-1/OnboardingChallengeKit/Sources/OnboardingChallengeKit/Screens/Detail/DetailViewController.swift

```diff
@@ -2,4 +2,32 @@
 
 import UIKit
 
-class DetailViewController: UIViewController {}
+class DetailViewController: UIViewController {
+    // MARK: Properties
+    private lazy var detailView: DetailView = {
+        return DetailView()
+    }()
+    
```

Comment:
I think that this implementation is a bit fragile. Here, we are defining the `DetailView`, creating a new `View` if it does not exists. However, we are not doing anything with it. We are trusting the `loadView()` implementation that it will call this property. However, if someone sets a different view in the `loadView()` the loaded view will be out of sync from this one and any change done on the `detailView` will be pointless.
Moreover, we won't ever notice that, even when running the app.

My suggestion is to change the property to cast the `self.view` to the right value:
```suggestion
    private lazy var detailView: DetailView {
        return self.view as! DetailView
    }
```

and to change the `loadView` as follows

```swift
func loadView() {
  self.view = DetailsView()
}
```

In this way, the only source of truth is the `loadView` method. If someone changes the view that is created and we run the app, the app will crash due to the force cast, but this is good because we want to fail quick in case of development errors.